app/lib/add_service.rb

```diff
@@ -65,7 +65,7 @@ def find_or_create_sub_service_request(line_item, service_request)
       end
     end
     sub_service_request = service_request.sub_service_requests.create(
-      organization_id: organization.id
```

Comment:
Because Rails can automatically assert IDs from class associations, you can shorten this statement to
```service_request.sub_service_requests.create(organization: organization, service_requester: @current_user)```